ee/clickhouse/views/person.py

```diff
@@ -46,8 +46,8 @@ def calculate_funnel_persons(self, request: Request) -> Dict[str, Tuple[list, Op
         if filter.funnel_viz_type == FunnelVizType.TRENDS:
             funnel_class = ClickhouseFunnelTrendsPersons
 
-        results = funnel_class(filter, team).run()
-        next_url = format_next_absolute_url(request, filter.offset, 100) if len(results) > 99 else None
+        results, should_paginate = funnel_class(filter, team).run()
+        next_url = format_next_absolute_url(request, filter.offset, 100) if should_paginate else None
```

Comment:
The "100" should be coming from the query param `LIMIT`? (with I guess a default like so^)